packages/ui/src/components/sign-in/steps/choose-session.tsx

```diff
@@ -0,0 +1,163 @@
+import { useClerk } from '@clerk/clerk-react';
+import { cva } from 'cva';
+import { Button } from 'react-aria-components';
+
+import { useAppearance } from '~/hooks/use-appearance';
+import { useDevModeWarning } from '~/hooks/use-dev-mode-warning';
+import { useDisplayConfig } from '~/hooks/use-display-config';
+import { useLocalizations } from '~/hooks/use-localizations';
+import * as Card from '~/primitives/card';
+import * as Icon from '~/primitives/icon';
+
+function getInitials({
+  firstName,
+  lastName,
+  identifier,
+}: {
+  firstName?: string | null;
+  lastName?: string | null;
+  identifier?: string | null;
+}): string {
+  let initials: string = '';
+  if (firstName) {
+    initials += firstName[0].toUpperCase();
+  }
+  if (lastName) {
+    initials += lastName[0].toUpperCase();
+  }
+  if (!initials && identifier) {
+    initials += identifier[0].toUpperCase();
+  }
+  return initials;
+}
+
+const sessionAction = cva({
+  base: 'text-left text-gray-11 hover:bg-gray-2 data-[focus-visible]:bg-gray-2 flex w-full items-center gap-x-3 px-10 py-4 text-base outline-none [--session-icon-opacity:0] data-[hovered]:[--session-icon-opacity:1]',
+});
+
+export function SignInChooseSession() {
+  const clerk = useClerk();
+  const { t } = useLocalizations();
+  const { layout } = useAppearance();
+  const renderDevModeWarning = useDevModeWarning();
+  const { branded } = useDisplayConfig();
+
+  const cardFooterProps = {
+    branded,
+    helpPageUrl: layout?.helpPageUrl,
+    privacyPageUrl: layout?.privacyPageUrl,
+    termsPageUrl: layout?.termsPageUrl,
+  };
+
+  const activeSessions = clerk.client.activeSessions;
+
+  return (
+    <Card.Root>
+      <Card.Content>
+        <Card.Header>
+          <Card.Title>{t('signIn.accountSwitcher.title')}</Card.Title>
+          <Card.Description>{t('signIn.accountSwitcher.subtitle')}</Card.Description>
+        </Card.Header>
+        <Card.Body>
+          <ul className='-mx-[--card-body-padding] -mb-[--card-body-padding] overflow-hidden rounded-b-[--card-content-rounded-b]'>
```

Comment:
Open to other opinions here, maybe we introduce a prop to remove the padding on card body? Left as is for now until we need it again and we can consider the best approach for full with body elements.